plugin-template

```diff
@@ -33,6 +33,7 @@ DEFAULT_SETTINGS = {
     'docker_fixtures': False,
     'docs_test': True,
     'issue_tracker': 'redmine',
+    'pulp_scheme': 'http',
```

Comment:
made `http` default, for now, so I can merge it.
After merging https://github.com/pulp/pulp-oci-images/pull/73 or https://github.com/pulp/pulp-oci-images/pull/84
we can change it to `https`